### DISCUSSION TITLE
万人血书，跪求赛博朋克2077

### DISCUSSION THREAD

**Discussion by newyorkthink** [Gamuxorg/bbs#413] [投递和请求游戏]
赛博朋克2077是标志性的游戏，
如果能把赛博朋克2077封装成appimage，
这使得linux游戏正营更加壮大。

cyberpunk : 天下无双

**Comment by nukemiko** [comment]
建议现在还是老老实实用Windows玩，现在的2.0.1版本Linux下性能极差，以前开光追都有几十帧，现在不开光追画质最低也就十几帧（都是拿3060跑的！）

**Comment by shouhuanxiaoji** [comment]
这个确实能运行，但是性能太差劲了
